.gitignore

```diff
@@ -24,6 +24,7 @@ speed-measure-plugin*.json
 *.sublime-workspace
 
 # IDE - VSCode
+.vscode
```

Comment:
why do you added this?

Reply:
To avoid my .vscode directory from being commited